runtime/wasm-bpf-rs/src/tests/mod.rs

```diff
@@ -0,0 +1,142 @@
+use crate::tests::pipe::ReadableWritePipe;
+
+use super::*;
+use std::fs::File;
```

Comment:
move the pipe implement out of test because ecli will needs it.

Reply:
Seems you are reviewing an outdated version of code....